### PR TITLE
SPCTR-2090 Slider Defaults and Bug fixes

### PR DESCRIPTION
**Pull request by patilvikasj**
### Description
Set defaults for slider block and bug fixes. 

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/slider/render.js [hunk 1]
```diff
@@ -18,6 +18,10 @@ const Render = ( props ) => {
 
 	const deviceType = useDeviceType();
 	const swiperRef = useRef();
+	const sliderWrapRef = React.useRef();
```
Comment (AbhijitNage123) [inline src/blocks/slider/render.js:21]: Any reason why we added `React.useRef()` instead of `useRef` directly?

**Review by AbhijitNage123 — CHANGES_REQUESTED** [review]

File: src/blocks/slider/render.js (hunk 1 above)
Comment (patilvikasj) [reply src/blocks/slider/render.js]: Right. We don't have to. Using `useRef` now.

**Review by AbhijitNage123 — APPROVED** [review]

File: includes/blocks/slider/attributes.php [hunk 2]
```diff
@@ -108,19 +112,19 @@
...
 		'boxShadowPositionHover'      => 'outset',
-		'arrowSize'                   => 30,
+		'arrowSize'                   => 16,
 		'arrowSizeTablet'             => '',
 		'arrowSizeMobile'             => '',
 		'arrowDistance'               => '',
 		'arrowDistanceTablet'         => '',
 		'arrowDistanceMobile'         => '',
-		'arrowPadding'                => '20',
+		'arrowPadding'                => '15',
```
Comment (bsf-zanev) [inline includes/blocks/slider/attributes.php:121]: Numbera shoudn't be in quotes.

**Review by bsf-zanev — CHANGES_REQUESTED** [review]
Please update the marked Integer Attributes in PHP.

File: includes/blocks/slider/attributes.php (hunk 2 above)
Comment (patilvikasj) [reply includes/blocks/slider/attributes.php]: Removed quotes now.

File: includes/blocks/slider/attributes.php [hunk 3]
```diff
@@ -108,19 +112,19 @@
...
 		'arrowPaddingTablet'          => '',
 		'arrowPaddingMobile'          => '',
-		'arrowColor'                  => '#000',
+		'arrowColor'                  => '#333333',
 		'arrowBgColor'                => '',
 		'verticalAlign'               => 'center',
-		'dotsMarginTop'               => '10',
+		'dotsMarginTop'               => '20',
```
[the remark this answers is not in the document]
Comment (patilvikasj) [reply includes/blocks/slider/attributes.php]: Removed quotes now.

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Pull request merged**